lib/src/allocation.dart

```diff
@@ -25,39 +25,31 @@ typedef PosixFree = void Function(Pointer);
 final PosixFree posixFree =
     stdlib.lookupFunction<PosixFreeNative, PosixFree>('free');
 
-typedef WinGetProcessHeapFn = Pointer Function();
-final WinGetProcessHeapFn winGetProcessHeap = stdlib
-    .lookupFunction<WinGetProcessHeapFn, WinGetProcessHeapFn>('GetProcessHeap');
-final Pointer processHeap = winGetProcessHeap();
+typedef WinCoTaskMemAllocNative = Pointer Function(Size cb);
```

Comment:
Size has been introduced to the Dart SDK in 2.17, so we need to bump the minimum SDK version to 2.17 and update the bot testing stable to 2.17.